src/lambda_services/api_service/api_service.py

```diff
@@ -1,36 +1,56 @@
-from string import ascii_lowercase, digits
+"""Generate unique job id and S3 tokens for each job."""
 from random import choices
+from string import ascii_lowercase, digits
 from typing import List
 import boto3
 
-# TODO 2020/17/20, Elvis - Establish specific logging format to be used in Lambda functions
+# TODO 2020/17/20, Elvis - Establish specific logging format to be used in
+#                  Lambda functions
+
 
 def create_s3_url(bucket_name: str, file_name: str, prefix_name: str) -> str:
-    object_name = f'{prefix_name}/{file_name}'
-    s3_client = boto3.client('s3')
+    """Format the READ section of the APBS input file.
```

Comment:
I assume the description here's a remnant from APBS code? If so, could we update it to say something like "Generate a presigned URL for file using S3 client"?

Reply:
Yes, copy/paste, forgot to update description.